src/common/utils/kaltura-params.ts

```diff
@@ -192,6 +194,31 @@ function addClientTag(url: string, productVersion?: string): string {
   return url;
 }
 
+/**
+ * @param {string} url - url
+ * @param {PKSourcesConfigObject} sources - sources
+ * @return {string} - the url with the seekFrom and clipTo appended in the query params
+ * @private
+ */
+function addStartAndEndTime(
+  url: string,
+  sources: PKSourcesConfigObject
+): string {
+  const seekFrom = Utils.Object.getPropertyPath(sources, 'seekFrom');
+  const clipTo = Utils.Object.getPropertyPath(sources, 'clipTo');
+  if (
+    seekFrom &&
```

Comment:
If you check typeOf ... == number, then "seekFrom &&" is redundant, since if its empty it will be undefined,